code/process_sequences.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Delong Tsway
+# dttsway@dons.usfca.edu
+# 2020-10-18
+
+# resources I consulted to build this code:
+# https://github.com/vsbuffalo/bioawk-tutorial/blob/master/README.md
+# https://opensource.com/article/17/2/command-line-tools-data-analysis-linux
+# https://www.poftut.com/awk-text-split-delimit-examples/
+# https://www.cyberciti.biz/faq/run-execute-sh-shell-script/
+# https://askubuntu.com/questions/420981/how-do-i-save-terminal-output-to-a-file
+
+FILE="/blast-db/sars-cov-2-seq-data/2020-10-15_ncbi_sars_cov_2_sequences.fasta.gz"
```

Comment:
Since the goal is to have the script take in the parameter from the command line, you should remove this, since it means the script will never work with any other file, since you set the file name inside the script.